lib/src/cmaf/ham/mapper/hls/mapHlsToHam.ts

```diff
@@ -180,7 +180,7 @@ function _videoPlaylistsToSwitchingSets(
 			segments: segments,
 			width: resolution.width,
 			height: resolution.height,
-			frameRate: playlist.attributes['FRAME-RATE'],
+			frameRateNum: playlist.attributes['FRAME-RATE'],
```

Comment:
Are we sure that frame rate in hls manifest always represent numerator?
In that case, we should infer denominator as 1 so as to have more information in ham model

Reply:
I just saw examples with one number in HLS. Not sure if it could be represented as 30/1 in HLS for example. But also I saw that today we are not using that value for HLS. Having that I mind, I think this should be covered in another ticket to figureout how to handle frameRate in HLS